src/components/SelectionList/BaseListItem.tsx

```diff
@@ -80,9 +82,9 @@ function BaseListItem<TItem extends ListItem>({
                         <PressableWithFeedback
                             accessibilityLabel={item.text}
                             role={CONST.ROLE.BUTTON}
-                            disabled={isDisabled}
+                            disabled={isDisabled || !isSelectable}
                             onPress={handleCheckboxPress}
-                            style={[styles.cursorUnset, StyleUtils.getCheckboxPressableStyle()]}
+                            style={[styles.cursorUnset, StyleUtils.getCheckboxPressableStyle(), !isSelectable && styles.cursorDisabled]}
```

Comment:
Should we apply `cursorDisabled` if `isDisabled` is true

Reply:
it is handled in `BaseGenericPressable.tsx` I think